homeassistant/components/autarco/quality_scale.yaml

```diff
@@ -68,14 +69,14 @@ rules:
       This integration cannot be discovered, it is a connecting to a service
       provider, which uses the users home address to get the data.
   docs-data-update: done
-  docs-examples: todo
-  docs-known-limitations: todo
+  docs-examples: done
```

Comment:
this one is not done imo, as it asks for ready-to-use automations in form of blueprints or YAML (and I couldn't find an open PR in docs)

Reply:
The only question is what kind of example I should add. I think we shouldn't force this in every integration, only if it's usefull.